x-pack/plugins/siem/public/components/navigation/tab_navigation/index.tsx

```diff
@@ -52,6 +58,7 @@ const TabNavigationContainer = styled.div`
   flex-direction: column;
   width: 100%;
   margin-top: -8px;
+  user-select: none;
```

Comment:
I toggled this off and the text still wasn't selectable (in Chrome) -- is this necessary?